app/helpers/field-type.js

```diff
@@ -0,0 +1,10 @@
+import { helper } from '@ember/component/helper';
+
+export function fieldType(params) {
```

Comment:
Please give a better name. `field-type` doesn't say anything about the helper at all. Very generic name.